# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/BufferTexture.h
# Commit: 509b05b741a2
# Full Hash: 509b05b741a2c5b27f467f9bd8c45bb1f32902da
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2022-07-12 21:56:41
# Regressor Bug: 1777426
# File Overlap Count: 1
# Description:
#   Bug 1777426 - Ensure MemoryTextureData frees memory when owned by RemoteTextureMap. r=aosmond,gfx-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D151422
# ==============================================================================

diff -r 2fdaa8eca97c -r 509b05b741a2 gfx/layers/BufferTexture.h
--- a/gfx/layers/BufferTexture.h	Tue Jul 12 06:56:20 2022 +0000
+++ b/gfx/layers/BufferTexture.h	Tue Jul 12 06:56:21 2022 +0000
@@ -96,6 +96,8 @@
                     gfx::BackendType aMoz2DBackend)
       : mDescriptor(aDescriptor), mMoz2DBackend(aMoz2DBackend) {}
 
+  virtual ~BufferTextureData() = default;
+
   BufferDescriptor mDescriptor;
   gfx::BackendType mMoz2DBackend;
 };